react/features/base/ui/components/native/Input.tsx

```diff
@@ -106,7 +119,7 @@ const Input = forwardRef<TextInput, IProps>(({
         onSubmitEditing?.(text);
     }, [ onSubmitEditing ]);
 
-    return (<View style = { [ styles.inputContainer, customStyles?.container ] }>
+    return (<View style = { [ styles.inputContainer, customStyles?.container ] as StyleProp<ViewStyle> }>
```

Comment:
do we need this casting?

Reply:
Kind of, yeah. The IDE was rigid about not having them.